lib/robots/dor_repo/gis_assembly/normalize_data.rb

```diff
@@ -31,29 +31,28 @@ def perform_work
           mods_filename = "#{rootdir}/metadata/descMetadata.xml"
           raise "normalize-data: #{bare_druid} is missing MODS metadata" unless File.size?(mods_filename)
 
-          format = GisRobotSuite.determine_file_format_from_mods mods_filename
-          raise "normalize-data: #{bare_druid} cannot determine file format from MODS" if format.nil?
+          raise "normalize-data: #{bare_druid} cannot determine file format" unless GisRobotSuite.media_type(cocina_object)
 
           # reproject based on file format information
-          if GisRobotSuite.vector?(format)
+          if GisRobotSuite.vector?(cocina_object)
             reproject_shapefile(filename, mods_filename, flags)
-          elsif GisRobotSuite.raster?(format)
+          elsif GisRobotSuite.raster?(cocina_object)
             projection = GisRobotSuite.determine_projection_from_mods mods_filename
             projection.gsub!('ESRI', 'EPSG')
             logger.debug "Projection = #{projection}"
-            filetype = format.split('format=')[1]
-            raise "normalize-data: #{bare_druid} cannot locate filetype from MODS format: #{format}" if filetype.nil?
 
-            case filetype
+            case GisRobotSuite.data_type(cocina_object)
```

Comment:
where's `data_type` defined?  i see a couple calls to it, but i don't see a `def` anywhere?

oh interesting, i see it declared in `lib/robots/dor_repo/gis_assembly/extract_iso19139.rb` on `main`, but not in this branch?  maybe i'm confused or missing something?  or maybe bad rebase somewhere?

Reply:
Should have been `data_format`. Fixed.